Mathlib/NumberTheory/Cyclotomic/Basic.lean

```diff
@@ -129,9 +129,10 @@ theorem trans (C : Type w) [CommRing C] [Algebra A C] [Algebra B C] [IsScalarTow
       refine ⟨algebraMap B C b, ?_⟩
       exact hb.map_of_injective h
     · exact ((isCyclotomicExtension_iff _ _ _).1 hT).1 hn
-  · refine adjoin_induction (((isCyclotomicExtension_iff T B _).1 hT).2 x)
+  · refine adjoin_induction (hx := ((isCyclotomicExtension_iff T B _).1 hT).2 x)
```

Comment:
Why are we having to use `(hx :=` here? Is this evidence that some implicit argument should be explicit? This PR certainly wasn't scared to add many new `_` inputs! (due to its nature).